### PR TITLE
Release v1.21.0

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: 1.21.0

## Release details

Packages:
- Adds support for running local ACN nodes
- Ports `ledger` and `http_client` connections as valory packages
- Ports `http`, `ledger_api` and `contract_api` protocols as valory packages
- Merges packages from `open-autonomy` and `open-aea` repo to prevent code duplication

Tests:
- Fixes flaky timeout tests
- Fixes flaky `DHT (ACN/Libp2p)` tests on windows
- Introduces test for assessing robustness of the ACN setup without agents

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side), from `develop`
- [x] Lint and unit tests pass locally and in CI
- [x] I have checked the fingerprint hashes are correct by running (`aea hash all` and `aea packages lock --check`)
- [x] I have regenerated the latest API docs
- [x] I built the documentation and updated it with the latest changes
- [x] I have added an item in `HISTORY.md` for this release
- [x] I bumped the version number in the `aea/__version__.py` file.
- [ ] I bumped the version number in every Docker image of the repo and published it. Also, I built and published them with tag `latest`  
      (check the READMEs of [`aea-develop`](../develop-image/README.md#publish) 
      and [`aea-user`](../user-image/README.md#publish))
- [ ] I have pushed the latest packages to the registry.
- [ ] I have uploaded the latest `aea` to PyPI.
- [ ] I have uploaded the latest plugins to PyPI.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Release History - open AEA
 
+## 1.21.0 (2022-09-28)
+
+Packages:
+- Adds support for running local ACN nodes
+- Ports `ledger` and `http_client` connections as valory packages
```
Comment (DavidMinarsch) [inline HISTORY.md:7]:
```suggestion
- Converts `ledger` and `http_client` connections to valory packages and syncs them with `open-autonomy` versions of the same packages
```

File: HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+- Ports `http`, `ledger_api` and `contract_api` protocols as valory packages
```
Comment (DavidMinarsch) [inline HISTORY.md:8]:
```suggestion
- Converts `http`, `ledger_api` and `contract_api` protocols to valory packages and syncs them with `open-autonomy` versions of the same packages
```

File: HISTORY.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+- Merges packages from `open-autonomy` and `open-aea` repo to prevent code duplication
```
Comment (DavidMinarsch) [inline HISTORY.md:9]: would remove

File: HISTORY.md [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+
+
+Tests:
+- Fixes flaky timeout tests
```
Comment (DavidMinarsch) [inline HISTORY.md:13]: where?

Comment (angrybayblade) [reply HISTORY.md:13]: https://github.com/valory-xyz/open-aea/pull/340

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**